Verslag/individual/michael/michael.tex

```diff
@@ -1,64 +1,315 @@
 
 \subsubsection{Detective measures}
 
-<Measures to detect if this application is installed>
-
+There are two large indicators to identify the malicious application. The first being the mis-matched icons. Exceptional about this is that the app doesn’t need to be running to identify it. The second indicator is during start-up. If there is a message in Arabic which pops up, it is a fake application. Since this application is not all that dangerous, there is no harm in this. However, for other applications this method is not recommended: it could lead to an application running malicious code. Preventive measures still apply at this point, such as verifying the legitimacy of an app by hash or size, or by checking the source of the application.
 \subsubsection{Reactive measures}
 
-<What to do if it is installed>
-
+If the application is already installed it is recommended to remove it. Whatever the case, do not run the application. In the case of this app, uninstalling the software would work, but in other cases applications might be harder to remove or have installed additional malicious software. In that case it might be a good idea to restore a back-up. Both iOS and Android support and even recommend this feature, by use of iCloud or Google Drive. If the device becomes inaccessible, it might be necessary to run the device in safe mode. This way it could still be booted and the application could be removed.
 \subsubsection{YARA rule set}
 
-<A YARA rule set created based on your findings>
+A simple way to filter this application is to create a YARA rule that detects if the string “www.farsroid.com” is present, and to specifically filter this app it is also useful to declare that the application size is below 40MB. Since the malicious version is smaller than the official one, and the official version is likely going to add updates such as extra levels or limited time events, the real app is likely to only get bigger. However, this condition could easily be removed so that the only requirement is that the string “www.farsroid.com” is present, to filter all apps made by this malicious developer.
+\begin{verbatim}
+
+rule filterfarsroidapps1
```

Comment:
deze yara gaat nooit werken, kijk ook bij mij hoe je de code met highlighting dr in kan krijgen :) dan hebben we dat namelijk hetzelfde

Reply:
Zal nog steeds niet werken maar je hebt wel zelfde highlighting